federator/src/lib/Federator.js

```diff
@@ -1,274 +1,365 @@
-const web3 = require('web3');
-const fs = require('fs');
-const TransactionSender = require('./TransactionSender');
-const CustomError = require('./CustomError');
-const BridgeFactory = require('../contracts/BridgeFactory');
-const FederationFactory = require('../contracts/FederationFactory');
-const AllowTokensFactory = require('../contracts/AllowTokensFactory');
-const utils = require('./utils');
+const web3 = require("web3");
```

Comment:
did you change much here apart from formatting and the utils stuff? the diff is kinda wonky and hard to review

Reply:
Actually, the biggest change was made doing the indentation of the file